package.json

```diff
@@ -11,6 +11,7 @@
       "packages/@local/*"
     ],
     "nohoist": [
+      "**/@types/css-font-loading-module",
```

Comment:
Fixes [linting](https://github.com/blockprotocol/blockprotocol/actions/runs/3312934753/jobs/5470229151#step:11:80)

If `@types/css-font-loading-module` is hoisted, `yarn lint:tsc` fails for Node-only packages that don’t include `DOM` lib in `tsconfig.json`. We could avoid that using `--skipLibCheck` instead, but that would make other (potentially impactful) issues unnoticed.